### PR TITLE
Refractor and Remove useless files 

### PR DESCRIPTION
**Pull request by espoirMur**
#### Description of changes made

> In this PR I removed the files that was useless to the application deployment .

It make simple the deployment using docker compose 

#### Type of change

> **Please select the relevant options**

- [ ] devops 

#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**

- After cloning the [repo](https://github.com/kisanola/nzembo.git)
- Check out this branch by `ref-add-simpler-stuff`
- make sure  , you have docker and docker-compose installed in your laptop 
- `docker-compose up -d `

- [ ] Unit

#### How should this be manually tested?

using 

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have ensured coding conventions for my code prior to submission
- [x] My changes generate no new warnings
- [x] I have added tests which prove the effectiveness of my work
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: docker-files/local/django/Dockerfile [hunk 1]
```diff
@@ -39,4 +35,3 @@ RUN chmod +x /start-flower
 
 WORKDIR /app
 
-ENTRYPOINT ["/entrypoint"]
```
Comment (tadomikikuto-bit) [inline docker-files/local/django/Dockerfile:42]: why remove this? we don't to make connection the db anymore?
where is the alternative cmd for this?

**Review by tadomikikuto-bit — CHANGES_REQUESTED** [review]

File: docker-compose-prod.yml [hunk 2]
```diff
@@ -15,10 +15,8 @@ services:
       - postgres
       - redis
     env_file:
-      - ./.envs/.production/.django
-      - ./.envs/.production/.postgres
+      - ./.env.production
```
[the remark this answers is not in the document]
Comment (espoirMur) [reply docker-compose-prod.yml:16]: Yeah, you are right man , it the same as the .env. sample but for production settings

File: docker-files/local/django/Dockerfile (hunk 1 above)
Comment (espoirMur) [reply docker-files/local/django/Dockerfile:42]: This is not making the connection to the database , it was just the entrypoint for the application.

File: config/settings/local.py [hunk 3]
```diff
@@ -1,72 +1,35 @@
...
     }
 }
-
-# EMAIL
-# ------------------------------------------------------------------------------
-# https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
 EMAIL_BACKEND = env(
     "DJANGO_EMAIL_BACKEND", default="django.core.mail.backends.console.EmailBackend"
 )
+INSTALLED_APPS = ["whitenoise.runserver_nostatic"] + INSTALLED_APPS  # noqa F405INSTALLED_APPS += ["debug_toolbar"]  # noqa F405
```
[the remark this answers is not in the document]
Comment (espoirMur) [reply config/settings/local.py:19]: I don't actually know..

File: config/urls.py [hunk 4]
```diff
@@ -43,16 +41,6 @@
...
-urlpatterns += [
-    re_path(r"^api/v1/song/", include("backend.song.api.urls", namespace="song")),
-    re_path(r"^api/v1/users/", include("backend.users.api.urls", namespace="users_api")),
-]
-urlpatterns += [
-    re_path(r'^swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
-    re_path(r'^swagger/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
-    re_path(r'^redoc/$', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
-]
-
```
[the remark this answers is not in the document]
Comment (espoirMur) [reply config/urls.py]: thanks for pointing out this man, I have corrected them .

**Review by espoirMur — COMMENTED** [review]
Lgtm

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+make-migrations-dev:
+	python manage.py makemigrations --settings=config.settings.local
+
+migrate-dev: 
+	python manage.py migrate --settings=config.settings.local
+
+test:
+	coverage run -m pytest -s --durations=0 .
+	coverage report
+	coverage html
```
Comment (tadomikikuto-bit) [inline Makefile:16]: it takes a lot of time to build the HTML especially when you have tested a lot of code. Can we simply have the report command?

**Review by tadomikikuto-bit — CHANGES_REQUESTED** [review]

**Review by tadomikikuto-bit — APPROVED** [review]

**Pull request merged**